patches/chrome-browser-net-stub_resolver_config_reader.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/net/stub_resolver_config_reader.cc b/chrome/browser/net/stub_resolver_config_reader.cc
+index 677f6c50fc602a1c5ba6ca52f4ea6496d7d60e7f..d736b6e06fcbee8fb17e3397cee86f3dbe09ad8e 100644
+--- a/chrome/browser/net/stub_resolver_config_reader.cc
++++ b/chrome/browser/net/stub_resolver_config_reader.cc
+@@ -353,6 +353,7 @@ SecureDnsConfig StubResolverConfigReader::GetAndUpdateConfiguration(
+ 
+   std::string doh_templates =
+       local_state_->GetString(prefs::kDnsOverHttpsTemplates);
++  BRAVE_GET_AND_UPDATE_CONFIGURATION
```

Comment:
This is to add DoH servers for support Unstoppable Domains. (And ENS in the future PR.)
These servers are controlled using its own prefs and not kDnsOverHttpsTemplates pref as the servers we added here are special and only applies to certain TLD which is different from user's global DoH provider settings.

Reply:
I would put this comment straight to the code :)

Reply:
Added in https://github.com/brave/brave-core/pull/7702/commits/0b940772d189bfa0d33bda335c0d3a7841c8b8f3